src/features/search/components/__tests__/SearchBoxRework.native.test.tsx

```diff
@@ -133,4 +133,38 @@ describe('SearchBox component', () => {
 
     expect(searchInput.props.value).toBe('')
   })
+
+  it('should not execute a search if input is empty', () => {
+    const { getByPlaceholderText } = render(<SearchBoxRework searchInputID={searchInputID} />)
+    const searchInput = getByPlaceholderText('Offre, artiste...')
+
+    fireEvent(searchInput, 'onSubmitEditing', { nativeEvent: { text: '' } })
+
+    expect(navigate).not.toBeCalledWith(
```

Comment:
idem : techniquement, on veut tester `not.toBeCalled()` plutôt que `not.toBeCalledWith(…)`